NDTensors/src/NDTensors.jl

```diff
@@ -41,32 +41,43 @@ include("aliasstyle.jl")
 include("abstractarray/set_types.jl")
 include("abstractarray/to_shape.jl")
 include("abstractarray/similar.jl")
+include("abstractarray/ndims.jl")
+include("abstractarray/fill.jl")
 include("array/set_types.jl")
 include("tupletools.jl")
-include("dims.jl")
-include("tensorstorage.jl")
-include("default_storage.jl")
+include("tensorstorage/tensorstorage.jl")
+include("tensorstorage/default_storage.jl")
 include("tensorstorage/similar.jl")
-include("tensor.jl")
+include("tensor/tensor.jl")
+include("dims.jl")
 include("tensor/set_types.jl")
 include("tensor/similar.jl")
 include("adapt.jl")
-include("generic_tensor_operations.jl")
-include("contraction_logic.jl")
+include("linearalgebra/generic_tensor_operations.jl")
+include("linearalgebra/contraction_logic.jl")
```

Comment:
Maybe these should go in a `tensoralgebra` folder instead? They don't quite fit as linear algebra.

Reply:
I was debating the name of the folder in my head. Should we have a `linearalgebra` and `tensoralgebra` folder or just one?

Reply:
I think having two makes sense. `linearalgebra` can be specific to operations that are specialized on order two tensors (i.e. `svd`, `qr`, `exp`, etc.).